GTM: Add info block (#5960)
segmentio/segment-docs: src/connections/destinations/catalog/google-tag-manager/index.md

3. In your Segment UI's destination settings, enter your Container ID (note: it should start with "GTM-"). You can find this in the Admin section of your [GTM dashboard](https://tagmanager.google.com/#/admin/){:target="_blank"}.
4. GTM loads on any pages where your Segment snippet is initialized and `analytics.page` is called in client-side JavaScript. Once you've turned on GTM through Segment, you can use Segment `track` events to populate the GTM `dataLayer`, and remove the GTML snippet from your page.

> info ""

> > Segment recommends that you load GTM through Segment rather than loading Segment inside of GTM. When you load Segment through GTM, it will limit our ability to help troubleshoot should you experience any technical issues. 

Reviewer: ```suggestion
> Segment recommends that you load GTM through Segment rather than loading Segment inside of GTM. When you load Segment through GTM, it limits Segment's ability to help you troubleshoot issues with your GTM integration. 
```

Reviewer: @sarahrudy just want to confirm before I commit this - would it be correct to say that "...it limits Segment's ability to help you troubleshoot issues with your GTM integration"? or would "GTM destination" be a better fit here

Author: @forstisabella i think in this case, you can probably just remove the "issues with your GTM integration" entirely if creates any confusion.
